test/test.transporters.js

```diff
@@ -71,6 +71,17 @@ describe('Transporters', function() {
     assert.equal(req.body, null);
   });
 
+it('should return errors as instances of Error', function (done) {
+    var google = require('../lib/googleapis');
+    var pagespeed = google.pagespeedonline('v1');
+
+    // intentional grammar mistake to trigger the error
+    pagespeed.pagespeedapi.runpagespeed({url: 'http://gogle.con'}, function (err) {
```

Comment:
But we just send the url to google and its servers tries to run the IP resolve, then they return the error when it fails. So we don't have any issue making the request on our end, but google's servers do.